shared_model/backend/protobuf/commands/impl/proto_command.cpp

```diff
@@ -95,6 +95,7 @@ namespace shared_model::proto {
         case iroha::protocol::Command::CommandCase::COMMAND_NOT_SET:
           assert(!"Unexpected command case.");
       };
+      std::abort();  // never return, suppress -Wreturn-type
     }()};
```

Comment:
```
default:
case iroha::protocol::Command::CommandCase::COMMAND_NOT_SET:
   std::abort();
```

Reply:
Maybe add some output with description.

Reply:
is is already there `          assert(!"Unexpected command case.");`

Reply:
```assert``` is invisible in release.

Reply:
And that is why I but direct std abort. I'd like to keep message in assert(). Or should I write another assert with message, which will not disappear in release build?

Reply:
done